xml/cap_admin_app_domains.xml

```diff
@@ -62,8 +62,9 @@
 
   &uaa-ca-cert-note;
 
-<screen>&prompt.user;helm install suse/cf \
---name susecf-scf \
+<screen>&prompt.user;kubectl create namespace scf
```

Comment:
Should this (and all other `helm install name` also be wrapped in </replaceable>?

Reply:
I just followed what the existing code had, but you're right they should. Will fix and update PR